rdflib/term.py

```diff
@@ -548,7 +556,7 @@ def __new__(cls, lexical_or_value, lang=None, datatype=None, normalize=None):
                 "per http://www.w3.org/TR/rdf-concepts/#section-Graph-Literal"
             )
 
-        if lang and not _is_valid_langtag(lang):
+        if lang is not None and not _is_valid_langtag(lang):
```

Comment:
I think this makes sense, though would like @gjhiggins to also sign off on it as this is what was in #1494.

The reason this makes sense to me is that `_is_valid_langtag` should be true for any lang tag, and if it is not true for a lang tag then the literal should not be constructed.

With the way it was `if lang and not _is_valid_langtag(lang):` there is range of values for `lang` for which `_is_valid_langtag` would be false, but for which Literal objects would be constructed  - and those literal objects would then have invalid language tags.

But I also think the typing here is very useful to prevent any confusion.